### PR TITLE
Style- added styles and positioning to both messaging features

### PR DESCRIPTION
**Pull request by tremitch504**

### CODE REVIEW

**Comment by jyoung3128** [comment]
Looks great Tre!

File: client/src/components/DirectMessage/Chat.jsx [hunk 1]
```diff
@@ -3,43 +3,59 @@ import React, {useState, useEffect, useContext} from 'react';
...
+    width: '10px',
+    height: '10px',
+    borderRadius: '50%',
+    backgroundColor: 'limegreen',
+    position: 'absolute',
+    top: '2px',
+    right: '2px'
+  };
   //to render the conversations
   return (
```
Comment (alexzmcginnis) [inline client/src/components/DirectMessage/Chat.jsx:39]: are you getting warnings here about not having a unique key?

File: client/src/components/DirectMessage/Chat.jsx [hunk 2]
```diff
@@ -3,43 +3,59 @@ import React, {useState, useEffect, useContext} from 'react';
...
-    <div className='chatonline'>
-      <div className='chatOnlineUser'>
-        <div className='chatUserName'>
-          {user.name}
-        </div>
-      </div>
-
+    <div className='chatOnline'>
+      {
+        onlineUsers.map( user => {
```
Comment (alexzmcginnis) [inline client/src/components/DirectMessage/Chat.jsx:42]: are you getting warnings here about not having unique keys?

File: client/src/components/DirectMessage/Chat.jsx [hunk 3]
```diff
@@ -3,43 +3,59 @@ import React, {useState, useEffect, useContext} from 'react';
...
-      } catch (err) {
-        console.warn(err);
-      }
-    };
-
-    getUser();
-  }, [conversation, currentUser]);
-
- 
+const ChatOnline = ({onlineUsers}) => {
```
Comment (alexzmcginnis) [inline client/src/components/DirectMessage/Chat.jsx:6]: ^i think the webpack is configured now to not need the regenerator-runtime

File: client/src/components/Messages/SidebarChat.jsx [hunk 4]
```diff
@@ -26,47 +26,57 @@ const Sidebar = () => {
...
+    position: 'absolute',
+    top: '2px',
+    right: '2px'
+  };
+  return (
+    <div className="chatOnline">
+      {
+        users.map( user => (
+          
+          <div className='chatOnlineUser' style={chatOnlineUser}>
```
Comment (alexzmcginnis) [inline client/src/components/Messages/SidebarChat.jsx:64]: i think this also might throw a warning for not having a unique key

**Review by alexzmcginnis — COMMENTED** [review]
looks good

**Pull request merged**